tools/api-markdown-documenter/src/RenderHtml.ts

```diff
@@ -66,14 +50,11 @@ export interface RenderDocumentsAsHtmlOptions
  *
  * @param documents - The documents to render. Each will be rendered to its own file on disk per
  * {@link DocumentNode.documentPath} (relative to the provided output directory).
- * @param renderConfig - Configuration for rendering {@link DocumentNode}s as HTML.
```

Comment:
All of this context is covered by property docs in `RenderDocumentsAsHtmlOptions`.